api/v1alpha1/consts.go

```diff
@@ -3,9 +3,11 @@ package v1alpha1
 import "time"
 
 const (
-	CertificateSigningRequestAPI      = "v1alpha1"
-	CertificateSigningRequestKind     = "CertificateSigningRequest"
-	CertificateSigningRequestListKind = "CertificateSigningRequestList"
+	APIVersionPrefix = "flightctl.io/"
```

Comment:
I believe K8s would call that the API group. Shall we leave out the trailing `/` here and instead add it in the function that constructs the group/version string?

Reply:
Fixed